app/src/main/java/org/xbmc/kore/Settings.java

```diff
@@ -132,6 +132,14 @@
     public static final String KEY_PREF_TVSHOWS_SHOW_WATCHED_STATUS = "tvshows_show_watched_status";
     public static final boolean DEFAULT_PREF_TVSHOWS_SHOW_WATCHED_STATUS = true;
 
+    // Filter disabled addons on addons list
+    public static final String KEY_PREF_ADDONS_FILTER_HIDE_DISABLED = "addons_filter_hide_disabled";
+    public static final boolean DEFAULT_PREF_ADDONS_FILTER_HIDE_DISABLED = false;
+
+    // Show disabled status on addon list
+    public static final String KEY_PREF_ADDONS_SHOW_DISABLED_STATUS = "addons_show_disabled_status";
```

Comment:
I do not think we need this as an option. Showing the disabled state should always be on IMO.

Reply:
Alright, I agree but tried to follow the movies convention (show watch status).